### PR TITLE
fix: pod resources schema

### PR DESCRIPTION
**Pull request by olevski**
This will address the following errors:
- https://sentry.dev.renku.ch/organizations/swiss-datascience-center/issues/4940/events/57bfd7f7151d4d5bb11d08825e054325/
- https://sentry.dev.renku.ch/organizations/swiss-datascience-center/issues/4939/events/9c1dce46c5c94a8a9ee3511f367b2082/

The issues were occurring because:
- now we also have an `ephemeral-storage` variable that we did not know before in the pod resources
- the timestamp that when the pod starts can be `null` sometimes instead of a timestamp

### CODE REVIEW

File: renku_notebooks/api/schemas.py [hunk 1]
```diff
@@ -77,14 +77,21 @@ def unnest_keys(self, data, **kwargs):
         return flatten_dict(data)
 
 
-class UserPodResources(Schema):
+class UserPodResources(
```
Comment (ableuler) [inline renku_notebooks/api/schemas.py:80]: I think this is equivalent to simply doing
```
    UserPodResources = Schema.from_dict(
        {
            "cpu": fields.Str(required=True),
            "memory": fields.Str(required=True),
            "ephemeral-storage": fields.Str(required=False),
        }
    )
```

**Review by ableuler — CHANGES_REQUESTED** [review]
A tiny bit of cosmetics... ;)

File: renku_notebooks/api/schemas.py (hunk 1 above)
Comment (olevski) [reply renku_notebooks/api/schemas.py:80]: Yeah I think you are right and it looks way better. I will change it.

**Review by ableuler — APPROVED** [review]
Thanks a lot for the quick turn around on this! 👍

**Pull request merged**